app/services/attribute_asserter.rb

```diff
@@ -44,9 +44,12 @@ def default_attrs
   end
 
   def add_bundle(attrs)
+    use_ascii = bundle.include?(:ascii)
     bundle.each do |attr|
       next unless VALID_ATTRIBUTES.include? attr
-      attrs[attr] = { getter: attribute_getter_function(attr) }
+      attrs[attr] = {
+        getter: use_ascii ? attribute_getter_function_ascii(attr) : attribute_getter_function(attr),
```

Comment:
It might be easier to read if this conditional logic is in a separate method - what do you think?